core/utils/src/main/kotlin/edu/stanford/spezi/core/utils/UiState.kt

```diff
@@ -0,0 +1,7 @@
+package edu.stanford.spezi.core.utils
+
+sealed interface UiState<out T> {
```

Comment:
It's a shame, I liked the approach because of the convention it offers with defined states. Even though it may not cover all cases, it could provide standardisation for a large number of them and introduce additional interfaces. i will remove it in an upcoming pr